src/Microsoft.AspNetCore.Mvc.ViewFeatures/AutoValidateAntiforgeryTokenAttribute.cs

```diff
@@ -27,7 +27,7 @@ public class AutoValidateAntiforgeryTokenAttribute : Attribute, IFilterFactory,
         /// </summary>
         /// <remarks>
         /// <para>
-        /// Filters are executed in an ordering determined by an ascending sort of the <see cref="Order"/> property.
+        /// Filters are executed in a sequence determined by an ascending sort of the <see cref="Order"/> property.
```

Comment:
@rynowak do you have any concerns about the verbiage here? It rings better to me.